frontend/js/main.js

```
@@ -2304,6 +2449,22 @@ class MataMonthlyDashboardManager {
       });
     }
 
+    // Gestionnaire pour le filtre par point de vente
+    const pointVenteFilter = document.getElementById('mata-point-vente-filter');
+    if (pointVenteFilter) {
+      pointVenteFilter.addEventListener('change', () => {
+        this.applyFilters();
+      });
+    }
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)